sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -403,6 +405,43 @@ export class ManagementClient extends LinkEntity<RequestResponseLink> {
     }
 
     try {
+      const { timeoutInMs } = sendRequestOptions;
+      let waitTimeForSendable = 1000;
+      if (
+        !this.link?.sender.sendable() &&
+        (timeoutInMs === undefined || timeoutInMs >= waitTimeForSendable)
+      ) {
+        internalLogger.verbose(
+          "%s Sender '%s', waiting for 1 second for sender to become sendable",
+          this.logPrefix,
+          this.name
+        );
+
+        await delay(waitTimeForSendable);
+
+        internalLogger.verbose(
+          "%s Sender '%s' after waiting for a second, credit: %d available: %d",
+          this.logPrefix,
+          this.name,
+          this.link?.sender.credit,
+          this.link?.session?.outgoing?.available()
+        );
+      } else {
+        waitTimeForSendable = 0;
```

Comment:
It is not clear to me what is the effect of zeroing this when it is not being used in the scope after?

Reply:
yes, it is not used here in copied code, but in messageSender.ts it is used in time calculation.  I've extracted into a util function.